README.md

```diff
@@ -1,110 +1,169 @@
 
-### Twilight aura
+![ChorusAura](https://user-images.githubusercontent.com/83953120/123547735-39b71a00-d762-11eb-8de4-ff3fb5e3fa24.gif)
 
-The **Twilight** aura creates colored fireflies  that will orbit around the player. The color of the fireflies can be changed to any color you want via the Illuminations dashboard. If the player moves, they will leave a trail of these fireflies. This aura is available to tier 1 donators.
+#### Golden Rod Aura
 
-![illuminations-1.0-twilight-aura](https://raw.githubusercontent.com/Ladysnake/Illuminations/main/README.assets/illuminations-1.0-twilight-aura.gif)
+This Aura will give you that little extra bit of sparkling, summoning Golden Rods all around you!
 
-### Ghostly aura
+![GoldenRodAura](https://user-images.githubusercontent.com/83953120/123548877-f317ee80-d766-11eb-9a04-d6bc27b73c67.gif)
 
-The **Ghostly** aura emanates little ghosts that fly upward and gradually appear and disappear. This aura is available to tier 2 donators.
+#### Shadowbringer Soul Aura
 
-![](https://raw.githubusercontent.com/Ladysnake/Illuminations/main/README.assets/illuminations-1.2-ghostly-aura.gif)
+This dark Aura will strike fear in your enemies as you are sourrounded by a storm of shady ashes!
 
-### Frost and Solar crowns
+![ShadowbringerSoulAura](https://user-images.githubusercontent.com/83953120/123550013-b13d7700-d76b-11eb-931f-19efd6dc509d.gif)
 
-3D overheads fit for a king or queen, that will either make you insanely cool, or astoundingly hot, depending on your choice.
+#### Sculk Tendrils Aura
 
-![illuminations-1.3-crowns](https://raw.githubusercontent.com/Ladysnake/Illuminations/main/README.assets/illuminations-1.3-crowns.png)
+Are you excited for 1.18? This Aura will summon Tendrils from the Deep Dark to follow you and grow around your feet.
 
-### Bloodfiend, Dreadlich and Mooncult crowns
+![SculkTendrilsAura](https://user-images.githubusercontent.com/83953120/123550344-5d339200-d76d-11eb-9803-e1d95d227bec.gif)
 
-*Designed by ArathainFarqoe!*
+#### Autumn Leaves Aura
 
-![image-20210113112423799](https://raw.githubusercontent.com/Ladysnake/Illuminations/main/README.assets/illuminations-1.4.1-crowns.png)
+A whirl of orange Leaves following your footsteps to bring that cosy autumn feeling to your world!
 
-### Chorus aura and crown
+![AutumnLeavesAura](https://user-images.githubusercontent.com/83953120/123550892-d0d69e80-d76f-11eb-9ffb-002bd73c5a8f.gif)
 
-![illuminations-1.4-choruscosmetics](https://raw.githubusercontent.com/Ladysnake/Illuminations/main/README.assets/illuminations-1.4-choruscosmetics.gif)
+### Overheads
 
-### Pride (and trans pride) heart
+Overheads are different hats that tier 2 donators can decide to wear.
 
-Available for free to anyone that registers for an Illuminations account. https://illuminations.uuid.gg/pride
+![Crowns](https://user-images.githubusercontent.com/83953120/123554291-95dc6700-d77f-11eb-8208-58205e4922a0.png)
```

Comment:
Too late now, but consider taking screenshots of Illuminations with Canvas Renderer, as that is one of the key points of this mod.